components/SubTrackDetails/SubTrackDetails.scss

```diff
@@ -0,0 +1,52 @@
+@import "topcoder/tc-includes";
+
+
+.SubTrackDetails{
+    
+    .heading {
+         height:20px;
+        
+        .title {
+            font-size: 15px;
+            line-height: 18px;
+            margin-left: 11px;
+            color: #FFFFFF;
+            float: left;
+            font-weight: bold;
+            font-family: "Roboto-Bold", Arial, Helvetica, sans-serif;
```

Comment:
If you switch the font-family name to just `'Roboto'`, the fonts should render correctly. Right now they are rendering as Arial. 
![image](https://cloud.githubusercontent.com/assets/5829188/14149153/a82ed7b2-f657-11e5-9ed8-140612b5cb82.png)